lib/api.js

```diff
@@ -9,7 +9,7 @@ import nativeCryptoLib from 'crypto'
 
 // 3_145_728 represent the maximum size for a transaction
 // 45_728 represent json tree size
-const MAX_FILE_SIZE = 3_145_728 - 45_728
+const MAX_FILE_SIZE = 3_000_000
```

Comment:
Why this shortcut ?

Reply:
I dont knew it was in megabytes and in megabites

Reply:
> Why this shortcut ?
what do you mean? . I added this to account for extra size .